Gems/GradientSignal/Code/Source/Editor/EditorGradientImageCreatorUtils.cpp

```diff
@@ -211,11 +215,16 @@ namespace GradientSignal::ImageCreatorUtils
             pixelFormat, pixelBuffer.data(), OIIO::AutoStride, OIIO::AutoStride, OIIO::AutoStride, WriteProgressCallback, saveDialog);
         if (!result)
         {
-            AZ_Error("EditorImageGradientComponent", result, "Failed to write out gradient image to path: %s", absolutePath.c_str());
+            AZ_Error("EditorImageGradientComponent", result, "Failed to write out gradient image to path: %s", tempSavePath.c_str());
         }
 
         outputImage->close();
 
+        // Now that we're done saving the temporary image, rename it to the correct file name.
+        [[maybe_unused]] auto moveResult = AZ::IO::SmartMove(tempSavePath.c_str(), absolutePath.c_str());
+        AZ_Error("EditorImageGradientComponent", moveResult,
+            "Failed to rename temporary image asset %s", tempSavePath.c_str(), absolutePath.c_str());
```

Comment:
You're passing both path strings into this message, but it only takes one.

Reply:
Whoops ty! It surprisingly didn't complain!

Reply:
nicely spotted!